packages/website/docs/Features/Javscript-library.md

```diff
@@ -5,8 +5,9 @@ sidebar_position: 3
 # Javascript Libraries
```

Comment:
idk if you care but this file is currently called `Javscript-library.md`

Reply:
I do care! fixed